### ISSUE TITLE
tap2deb/tap2rpm tests should not produce deprecation warnings

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#8095]
|[<img alt="hawkowl's avatar" src="https://avatars.githubusercontent.com/u/3307100?s=50" width="50" height="50">](https://github.com/hawkowl)| @hawkowl reported|
|-|-|
|Trac ID|trac#8095|
|Type|enhancement|
|Created|2015-11-07 11:37:16Z|
|Branch|https://github.com/twisted/twisted/tree/tap2nodeprec-8095|

Part of [#8092](https://github.com/twisted/twisted/issues/8092)

<details><summary>Searchable metadata</summary>

```
trac-id__8095 8095
type__enhancement enhancement
reporter__hawkowl hawkowl
priority__normal normal
milestone__None None
branch__branches_tap2nodeprec_8095 branches/tap2nodeprec-8095
branch_author__hawkowl hawkowl
status__closed closed
resolution__fixed fixed
component__core core
keywords__None None
time__1446896236995422 1446896236995422
changetime__1447413778008372 1447413778008372
version__None None
owner__hawkowl hawkowl

```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="hawkowl's avatar" src="https://avatars.githubusercontent.com/u/3307100?s=50" width="50" height="50">](https://github.com/hawkowl)<a name="note_1"></a>|@hawkowl commented|
|-|-|

(In [46103]) Branching to tap2nodeprec-8095.

**Comment by twisted-trac** [comment]
|[<img alt="hawkowl's avatar" src="https://avatars.githubusercontent.com/u/3307100?s=50" width="50" height="50">](https://github.com/hawkowl)<a name="note_2"></a>|@hawkowl commented|
|-|-|

Tests are looking reasonable. Please review.

**Comment by twisted-trac** [comment]
|[<img alt="adiroiban's avatar" src="https://avatars.githubusercontent.com/u/204609?s=50" width="50" height="50">](https://github.com/adiroiban)<a name="note_3"></a>|@adiroiban set owner to @hawkowl|
|-|-|

Changes looks good. Thanks

Can we filter just DeprecationsWarnings ?


Please check my comment and merge.

Thanks!

**Comment by twisted-trac** [comment]
|[<img alt="hawkowl's avatar" src="https://avatars.githubusercontent.com/u/3307100?s=50" width="50" height="50">](https://github.com/hawkowl)<a name="note_4"></a>|@hawkowl set status to `closed`|
|-|-|

(In [46180]) Merge tap2nodeprec-8095: Prevent the tap2deb/tap2rpm tests from raising warnings

Author: hawkowl
Reviewer: adiroiban
Fixes: [#8095](https://github.com/twisted/twisted/issues/8095)
